exporter/collector/config.go

```diff
@@ -180,6 +180,9 @@ type LogConfig struct {
 	// service.name, service.namespace, and service.instance.id resource attributes into the Cloud Logging LogEntry labels.
 	// Disabling this option does not prevent resource_filters from adding those labels. Default is true.
 	ServiceResourceLabels bool `mapstructure:"service_resource_labels"`
+	// ErrorReportingType enables automatically parsing error logs to a json payload containing the
+	// type value for GCP Error Reporting. See https://cloud.google.com/error-reporting/docs/formatting-error-messages#log-text.
+	ErrorReportingType bool `mapstructure:"error_reporting_type"`
```

Comment:
nit: it isn't clear from the name what it does.  Prefer send_errors_to_error_reporting, or something that implies what the effect is.

Reply:
afaict, adding this attribute doesn't send them "to" error reporting. they're still sent to cloud logging, just reformatted to have a special attribute that Error Reporting picks up. I'm not entirely clear on the ux from that point, will talk to some of the folks who work on ER

Reply:
Sounds good.  Thanks for checking!